__tests__/storage/memory_store.test.js

```diff
@@ -262,6 +262,125 @@ test('utxo methods', async () => {
   expect(buf[0].txId).toEqual('tx02');
 });
 
+test('locked utxo methods', async () => {
+  const tsFromDate = (date) => Math.floor(date.getTime() / 1000);
+  const tsBefore = new Date('2023-03-21T11:00:00');
+  const tsCurrent = new Date('2023-03-21T12:00:00');
+  const tsAfter = new Date('2023-03-21T13:00:00');
+
+  function getLockedUtxo(txId, address, timelock, height, value, token, token_data) {
+    return {
+      index: 0,
+      tx: {
+        tx_id: txId,
+        height,
+        version: 1,
+        timestamp: tsFromDate(tsBefore),
+        is_voided: false,
+        outputs: [
+          {
+            value,
+            token_data,
+            token,
+            spent_by: null,
+            decoded: {
+              type: 'P2PKH',
+              address,
+              timelock,
+            }
+          },
+        ],
+      },
+    };
+  }
+
+  function getUtxoFromLocked(lutxo) {
+    const { tx, index } = lutxo;
+    const { outputs } = tx;
+    const output = outputs[index];
+    const { decoded } = output;
+    const { address, timelock } = decoded;
+    return {
+      txId: tx.tx_id,
+      index,
+      token: output.token,
+      address,
+      value: output.value,
+      authorities: 0,
+      timelock,
+      type: tx.version,
+      height: tx.height,
+    };
+  }
+
+  jest
+   .useFakeTimers()
+   .setSystemTime(tsCurrent);
+
+  const store = new MemoryStore();
```

Comment:
Can you do the same tests for the leveldb store?

Reply:
Done.